pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 90
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
```

Comment:
```suggestion
target-version = ['py38', 'py39', 'py310', 'py311']
```

Reply:
we still does support python 3.7 at least now. probably would be dropping it later this year